src/app/services/validators/schema_validator_service.py

```diff
@@ -0,0 +1,47 @@
+from exception.exceptions import ValidationException
+
+
+class SchemaValidatorService:
```

Comment:
Hey Isaiah, could we add a doc string to the class and describe why it is used over pydantic please. (Really to convey our discussions on pydantic vs base models as there is a lot of switching between the two over the last two months).  Chances are developers might miss it but think would be to document in validator for future reference.

Reply:
Have added doc string to the class explaining why dict is used over Pydantic model and the usage of this validator